v1/src/main/java/com/google/cloud/teleport/templates/common/JdbcConverters.java

```diff
@@ -222,9 +226,16 @@ public TableRow mapRow(ResultSet resultSet) throws Exception {
                 DATE_FORMATTER.format(resultSet.getDate(i).toLocalDate()));
             break;
           case "datetime":
-            outputTableRow.set(
-                getColumnRef(metaData, i),
-                DATETIME_FORMATTER.format((TemporalAccessor) resultSet.getObject(i)));
+            if (driverClassName.get().equals(MSSQL_DRIVER_CLASS)) {
```

Comment:
This is a good change -- I think you can replace this if with `if (resultSet.getTimestamp(i) instanceof Timestamp)` so we generalize this for other drivers (that may happen to also return java.sql.Timestamp)

Reply:
Done, I like this change more